charts/dbildungs-iam-server/values.yaml

```diff
@@ -124,7 +124,9 @@ backend:
     interval: 30m
     endpoints:
       - port: 'web'
-  extraEnvVars: []
+  extraEnvVars:
```

Comment:
is there a reason why you use this extraEnvVars? I think this setting would be easier to understand if this value is also placed in the configmap. I think i myself added the extraEnvVars, even though i can currently not think of a reason why we would need it :D Might get deleted as well

Reply:
It made sense to me to have it _separate_, because the setting is fairly low-level/does not have much to do with the app itself and probably won't be changed often.
I follow you reasoning though. I don't mind either way, so I will move it into the ConfigMap :)